packages/react-hooks/src/hooks/force-update.ts

```diff
@@ -0,0 +1,6 @@
+import {useState, useCallback} from 'react';
+
+export function useForceUpdate() {
```

Comment:
Can we simply do this: 
```
const [,setState] = useState();
return setState;
```

Reply:
Answered on Slack, but copying my reply here for transparency.

`setState(undefined)` wouldn't work because we absolutely need to provide a new value to trigger a new render. At best, as explained by this issue https://github.com/facebook/react/issues/17474, setting the state with the same value might trigger en 2nd render, but not more than that because the component is not even supposed to rerender when the value is the same.

Another way to write it is with `useReducer`, but I thought it would be best to keep it simple with `useState`.
```
const [, forceUpdate] = useReducer(() => ({}), {});
return forceUpdate;
```

WDYT? Prefer the reducer way?